source/_patterns/03-organisms/sections/modal/_modal.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 // Hides the dialog window. This attr is added by the A11yDialog script.
-.jcc-modal [aria-hidden="true"] {
+.jcc-modal[aria-hidden="true"] {
   @include u-display(none);
```

Comment:
@slowbot Note: This fixes a bug I noticed in Safari. You can see it here: 
- https://confident-allen-d061ed.netlify.com/?p=organisms-modal
- https://confident-allen-d061ed.netlify.com/?p=organisms-timeline

Looks like this:
<img width="1296" alt="Screen Shot 2020-01-08 at 1 12 00 PM" src="https://user-images.githubusercontent.com/60979/72009308-9ef01600-3223-11ea-94ea-a6af501efa35.png">